.github/workflows/flatpak.yml

```diff
@@ -3,10 +3,10 @@ name: Flatpak (Experimental)
 on:
   push:
     paths-ignore: ['**.md']
-    branches: [master]
+    branches: [disabled-master]
```

Comment:
Is this just to disable this workflow?

Reply:
Yes it was remade into the `main.yml` from what I understood.

Reply:
There seems to be no other way in the YAML syntax to simply disable a workflow while leaving the file intact.  We can disable the workflow from the GItHub Actions UI, but this seems fine too.